dataactvalidator/validation_handlers/validator.py

```diff
@@ -210,6 +219,9 @@ def validate_file_by_sql(job, file_type, short_to_long_dict, batch_results=False
     file_id = FILE_TYPE_DICT[file_type]
     rules = sess.query(RuleSql).filter_by(file_id=file_id, rule_cross_file_flag=False)
     errors = []
+    rules_run = 0
+    progress_check = 2 if file_type != 'fabs' else 5
```

Comment:
Can we include a comment here to explain this? Like how we're checking every 2 for DABS and every other for FABS?